uber/site_sections/registration.py

```diff
@@ -147,6 +149,69 @@ def history(self, session, id):
                                .order_by(Tracking.when).all()
         }
 
```

Comment:
Minor quibble about names, but instead of ``id`` and ``watchlist_id`` I would prefer ``attendee_id`` and ``watchlist_id``, which is consistent with how we usually handle pages which accept two different types of ids (e.g. we have handlers with things like ``attendee_id`` and ``job_id`` on some of the shift management forms).